### PR TITLE
feat(balance): Misc Repair Kit Knives

### PR DESCRIPTION
**Pull request by yay855**
Made the misc repair kit able to be made with any number of civilian-grade knives with fine cutting.

## Purpose of change

The misc repair kit wasn't craftable innawoods, nor was it too easily craftable otherwise; it needed an xacto knife, which are relatively uncommon.

## Describe the solution

The misc repair kit can now be crafted using any tool knife with fine cutting, not including scalpels.

### CODE REVIEW

File: data/json/recipes/other/tools.json [hunk 1]
```diff
@@ -772,7 +772,20 @@
...
     "components": [
-      [ [ "xacto", 1 ] ],
+      [
+        [ "xacto", 1 ],
+        [ "knife_steak", 1 ],
+        [ "knife_paring", 1 ],
+        [ "knife_chef", 1 ],
+        [ "knife_carving", 1 ],
+        [ "knife_bread", 1 ],
+        [ "xacto", 1 ],
```
Comment (chaosvolt) [inline data/json/recipes/other/tools.json:782]: Already specified earlier so
```suggestion
```

Comment (Lamandus) [reply data/json/recipes/other/tools.json:782]: how can you suggest a removal without adding a new line?

Comment (chaosvolt) [reply data/json/recipes/other/tools.json:782]: I clicked the suggestion button than deleted the text inside the ```s.

**Comment by chaosvolt** [comment]
The other thing I realized is the kit itself will still count as having fine cutting 2: https://github.com/cataclysmbnteam/Cataclysm-BN/blob/main/data/json/items/tool/workshop.json#L593

Though, it also won't be able to handle the fact that all those knives will have wildly varying levels of butchering quality so hmm, plus mods have no real sane way to allow smoothly injecting their own knives into the recipe (granted this is also a problem with toolboxes already so eh)...bleh this is less of a simple thing to fix than I thought. :/

**Comment by yay855** [comment]
> The other thing I realized is the kit itself will still count as having fine cutting 2: https://github.com/cataclysmbnteam/Cataclysm-BN/blob/main/data/json/items/tool/workshop.json#L593
> 
> Though, it also won't be able to handle the fact that all those knives will have wildly varying levels of butchering quality so hmm, plus mods have no real sane way to allow smoothly injecting their own knives into the recipe (granted this is also a problem with toolboxes already so eh)...bleh this is less of a simple thing to fix than I thought. :/

It really is. TBH we might be better off just making scissors have fine cutting 2 and letting _those_ be the new craftable ingredient. A good sharp pair of scissors can be pretty useful for cutting.

**Comment by Lamandus** [comment]
> > The other thing I realized is the kit itself will still count as having fine cutting 2: https://github.com/cataclysmbnteam/Cataclysm-BN/blob/main/data/json/items/tool/workshop.json#L593
> > Though, it also won't be able to handle the fact that all those knives will have wildly varying levels of butchering quality so hmm, plus mods have no real sane way to allow smoothly injecting their own knives into the recipe (granted this is also a problem with toolboxes already so eh)...bleh this is less of a simple thing to fix than I thought. :/
> 
> It really is. TBH we might be better off just making scissors have fine cutting 2 and letting _those_ be the new craftable ingredient. A good sharp pair of scissors can be pretty useful for cutting.

another problem with that. fine cutting 2 is for dissecting corpses, sure, the Grimm brothers loved to cut up wolf's stomach with scissors, it isn't a good pseudo scalpel.


my opinion:

- remove any knife from the recipe.
- add scissors as its counterpart
- change quality of the repair kit to match the scissors

**Comment by yay855** [comment]
There, I made the misc repair kit require scissors instead and not have the fine cutting quality.

**Review by chaosvolt — APPROVED** [review]
This is an improvement since scissors can be crafted and x-acto blades can't (though I still kinda wouldn't mind fixing that anyway), but I guess one of these days I oughta mainline the makeshift repair kit from MST Extra for an innawood option...

**Pull request merged**